syntax = "proto3";

import "proto/bank/account.proto";

package bank;

//option java_package = "com.proto.bank";
//option java_multiple_files = true;

message User {
  string first_name = 1;
  string last_name = 2;
  int32 id = 3;
  Account account = 4;
}

message UserRequest {
  User user = 1;
}

message UserResponse {
  string result = 1;
}

service UserService {
  // Unary
  rpc User(UserRequest) returns (UserResponse) {};
}